hipcheck/src/plugin/manager.rs

```
@@ -146,14 +249,29 @@ impl PluginExecutor {
 			spawn_args.push("--port");
 			spawn_args.push(port_str.as_str());
 
+			// pass logging level for plugin
+			let plugin_title = plugin
+				.name
+				.split_once('/')
+				.map(|(_, part)| part)
+				.unwrap_or(&plugin.name);
+			let plugin_path = format!("plugin::{}", plugin_title);
+			let plugin_log_level = match self.get_log_level_for_module(&plugin_path) {
+				Some(level) => level,
+				None => String::from("OFF"), // Default to "OFF"
```

What's the motivation for defaulting to `OFF`? If I understand correctly, this suppresses all logging, even errors. (I have yet to try these changes in my environment, so I may be missing some subtlety)

If this does suppress all errors from plugins by default, I think that it would be surprising to users to not be informed when plugins fail and why, since it effects how they interpret the results, and can point to problems with their environment or source data.

This is especially unfortunate for debugging, given that currently, the only way for plugins to report errors other than by logging (now tracing) is the `UnspecifiedQueryState` error that gives no information. (Though I'm working on a proposal to improve that situation)

---

Based on our discussion it appears we are defaulting plugins to OFF, but looking to find ways to help plugins communicate when targets are invalid.

---

Updated to default plugins to error if no relevant HC_LOG var is set.